errai-data-binding/src/main/java/org/jboss/errai/databinding/rebind/DataBindingUtil.java

```diff
@@ -464,8 +461,56 @@ public static boolean isBindableType(final MetaClass type) {
         }
       }
     }
+  }
 
-    return bindableTypes;
+  private static void addConfiguredBindableTypes(final Set<MetaClass> bindableClasses, String value) {
```

Comment:
It seems the only difference between this method and the one below is the error message. Could we possibly refactor these to use a single helper method with an error message parameter?